### ISSUE TITLE
Omega 21.x

### ISSUE THREAD

**Issue by thezoggy** [fhriley/kodi-headless-novnc#10]
Do you plan to start offering an Omega tag for when 21.x hits RC or only after final?

**Comment by fhriley** [comment]
Yes, when omega RC builds come out I will add an omega image.

**Comment by thezoggy** [comment]
great to hear, i just got a new tv that can do dolby vision.. so im needing 21.x to do dv in mkv (newer ffmpeg)

**Comment by aevans0001** [comment]
is there a way to change the port 8000?  my system has something already registered.  I tried clearing that but i dont really know what it is.  However, i was able to try to connect to the kodi docker and it shows


`unversioned API: requested URI not found`

```
version: "3"

services:
  kodi:
   image: fhriley/kodi-headless-novnc:Nexus
   network_mode: "container:wireguard"
   restart: always
   init: true
     #  ports:
     # "5900:5900/tcp"
     # "8000:8000/tcp"
     # "8080:8080/tcp"
     # "9090:9090/tcp"
     # "9777:9777/udp"
   environment:
      - KODI_UID=1000
      - KODI_GID=1000
      - TZ=America/Chicago
   volumes:
      - /docker-compose/kodi/config:/config
      - /mnt/storage/media:/media
      - /mnt/storage/media/Downloads:/downloads
```


I am able to access it from 8080 but would like to see it in action on 8000.  also how do I add file manager into it.

**Comment by fhriley** [comment]
@thezoggy The `Omega` tag is now available and points to `21.0b1`.

**Comment by thezoggy** [comment]
upgraded today to your omega tag, everything working great! thanks again
